src/transcoding/transcoding.h

```diff
@@ -221,7 +221,7 @@ class TranscodingProfileList {
     std::shared_ptr<TranscodingProfileMap> get(const std::string& sourceMimeType);
     const std::map<std::string, std::shared_ptr<TranscodingProfileMap>>& getList() { return list; }
     std::shared_ptr<TranscodingProfile> getByName(const std::string& name, bool getAll = false);
-    int size() const { return list.size(); }
+    int size() const { return int(list.size()); }
```

Comment:
try to change the return type to size_t

Reply:
In src/file_request_handler.cc there are two lines

```
if (res_id != -1)
```

which would break if this is done as size_t cannot have negative numbers.